tools/interactive/interactivetool_jupyter_notebook.xml

```diff
@@ -63,8 +76,13 @@
                     help="This option is useful in workflows when you just want to execute a notebook and not dive into the webfrontend."/>
             </when>
         </conditional>
-        <param name="input" type="data" optional="true" label="Include data into the environment"/>
-
+        <repeat name="input_repeats" title="Input Dataset(s)" >
```

Comment:
I think this is too inflexible ... collections could also be taken etc ...
And further the real problems are the outputs, as they are highly flexible, inputs can probably be taken by get() even in a workflow.